### DISCUSSION TITLE
smooth_shape	API is not working

### DISCUSSION THREAD

**Discussion by arvindg007** [CGAL/cgal#7159] [Q&A]
I tried smooth_shape (https://doc.cgal.org/latest/Polygon_mesh_processing/group__PMP__meshing__grp.html#ga57fa999abe8dc557003482444df2a189)
 API which takes FaceRange as input, but it is not updating mesh at all. 
On debug I noticed that it fails because smoother.solve_system returns false.  Internally solver is giving NumericalIssue error.

whereas for same face ranges other API works.  I want to use smooth_shape for smoothening some faces of the mesh. 

Any idea what could be the reason?

**Comment by MaelRL** [comment]
What do you mean by "other API"? Can you please provide the exact input (data + parameters)?
